mkosi.postinst.chroot

```diff
@@ -40,6 +40,10 @@ cp "$SRCDIR/factory/etc/nsswitch.conf" /etc/nsswitch.conf
 # Remove to make TEST-73-LOCALE pass on Ubuntu.
 rm -f /etc/default/keyboard
 
+# OpenSUSE disables all controller delegation for the user services. Remove the dropin to make TEST-55-OOMD
+# pass on OpenSUSE.
+rm -f /usr/lib/systemd/system/user@.service.d/20-defaults-SUSE.conf
```

Comment:
maybe mask the drop in in the test case instead? so that the generic image is more similar to a suse image

Reply:
It's rather complicated to fix up the controllers after boot.

Reply:
What about command line/credential when booting for the test case?

Reply:
We don't have a command line or credential for masking dropins AFAIK.

Reply:
You added support for passing arbitrary drop-ins through credentials, and passing an empty file should do the trick?

Reply:
@YHNdnzj You think so? We need to symlink to /dev/null to mask a dropin, I don't think that logic is in place yet?

Reply:
masking can also be done with an empty file, doesn't need to be a symlink